WBooks/Views/Book Detail/BookDetailViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  BookDetailViewController.swift
+//  WBooks
+//
+//  Created by Salvador Carnelutti on 03/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import WolmoCore
+
+class BookDetailViewController: UIViewController {
+    private var _view: BookDetailView = BookDetailView.loadFromNib()!
+    private var _commentRepository: CommentRepository = CommentRepository()
+    private var _viewModel: BookViewModel
+    private var _bookDetailViewModel: BookDetailViewModel
+    
+    init(bookVM: BookViewModel) {
+        _viewModel = bookVM
+        _bookDetailViewModel = BookDetailViewModel()
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    override func loadView() {
+        view = _view
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        _view.backgroundColor = UIColor(red: 0.92, green: 0.96, blue: 0.98, alpha: 1.0)
+        _view.detailTable.dataSource = self
+        _view.detailTable.delegate = self
+        _view.detailTable.register(cell: BookDetailCell.self)
+        navigationItem.title = "BOOK DETAIL"
+        setCurrentBook()
+        _commentRepository.fetchComments(
+            bookID: _viewModel.id,
+            onSuccess: { [weak self] comments in
+                self?._bookDetailViewModel.comments = comments.map { CommentViewModel(comment: $0) }
+                print(comments)
+                //self?._viewModel.books = books.map { BookViewModel(book: $0) }
+                self?._view.detailTable?.reloadData()
+            }, onError: { error in
+                print(error)
+        })
+    }
+}
+
+extension BookDetailViewController: UITableViewDataSource, UITableViewDelegate {
+    func setCurrentBook() {
+        _view.lblBookTitle.text = _viewModel.title
+        _view.lblStatus.text = _viewModel.status.capitalized
+
+        _view.lblAuthor.text = _viewModel.author
+        _view.lblYear.text = _viewModel.year
+        _view.lblGenre.text = _viewModel.genre
+        let imageUrl = URL(string: _viewModel.image)
+        let data = try? Data(contentsOf: imageUrl ?? "http://wolox-training.s3.amazonaws.com/uploads/6942334-M.jpg")
+        if let imageData = data {
+            let bookImage = UIImage(data: imageData)
+            _view.imgBookCover.image = bookImage
```

Comment:
Is there a reason why you split this into two lines? I think you can do:
`_view.imgBookCover.image = UIImage(data: imageData)`

Reply:
Same as the other one, I'd swap the order of some lines and give some space:

```
        let imageUrl = URL(string: _viewModel.image)
        let data = try? Data(contentsOf: imageUrl ?? "http://wolox-training.s3.amazonaws.com/uploads/6942334-M.jpg")

        _view.lblBookTitle.text = _viewModel.title
        _view.lblStatus.text = _viewModel.status.capitalized
        _view.lblAuthor.text = _viewModel.author
        _view.lblYear.text = _viewModel.year
        _view.lblGenre.text = _viewModel.genre

        if let imageData = data {
            _view.imgBookCover.image = UIImage(data: imageData)
        }
```

Reply:
When you tell me to give some space, what do you mean?